tox.ini

```diff
@@ -50,8 +51,7 @@ deps:
     sphinx
 
 [flake8]
-enable-extensions = U4
 exclude = conf.py
-hang-closing = True
 jobs = 1
 max-line-length = 79
+ignore = E123, E133, W503, W504
```

Comment:
Reason for ignoring all these:

`E123` -- this is a too-narrow interpretation of [PEP8's "closing brace"](https://www.python.org/dev/peps/pep-0008/#:~:text=The%20closing%20brace) guide.
`E133` -- likewise
`W503` & `W504` -- these are exclusive to one another. If you fulfill W503, you'll violate W504, and vice versa. Also both of them are too-narrow interpretation of [PEP8's "Line break before/after binary operator"](https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator)

(Note, the most recent recommendation is actually `W504`, but that causes a LOT of `qa` issues. Let's fix that in a separate PR)

Reply:
FWIW, I'd be happy to adopt `black` for code formatting, because auto formatting is 👍